internal/postrs/pos_verifier.go

```diff
@@ -0,0 +1,69 @@
+package postrs
+
+// #cgo LDFLAGS: -lpost
+// #include <stdlib.h>
+// #include "post.h"
+import "C"
+
+import (
+	"errors"
+	"unsafe"
+)
+
+type VerifyPosOptions struct {
+	fromFile *uint32
+	toFile   *uint32
+	fraction float64
+}
+
+var ErrInvalidPos = errors.New("invalid POS")
+
+type VerifyPosOptionsFunc func(*VerifyPosOptions) error
```

Comment:
Could we also add `WithLogger` and make `setLogCallback` private again? Otherwise Pos Verification API would differ from all the others in this module in how to setup logging.

Reply:
Hmm I'm not sure how to approach it best. `func WithLogger()` is already "taken" for opts in api.go. Should we start to divide into subpackages at this point?

Reply:
The quick way around it is to call it `WithVerifyPosLogger` or similar. I would like to avoid splitting the package just because of that.

Alternativelly we could have a single `option` struct that is shared between this function and others in the package. Then we also only need one `WithLogger` functional option for all of them. Since this is an `internal` package I also think this would be OK, since it cannot be accidentally used incorrectly from another module.